packages/ui-markdown-editor/src/lib/components/Span/HorizontalRule.js

```diff
@@ -1,12 +1,58 @@
+import React from 'react';
 import styled from 'styled-components';
+import PropTypes from 'prop-types';
 
-const HorizontalRule = styled.div`
+const HorizontalRuleWrapper = styled.div`
   overflow: visible; /* For IE */
   padding: 0;
   border: none;
-  border-top: medium double #333;
-  color: #333;
   text-align: center;
+  display: grid;
+  grid-template-columns: auto max-content auto;
+  user-select: none;
 `;
 
+const HorizontalLine = styled.div`
+  overflow: visible; /* For IE */
+  display: grid;
+  align-self: center;
+  padding: 0;
+  height: 1px;
+  background-color: #979797;
+  text-align: center;
+  user-select: none;
+`;
+
+const PageBreak = styled.span`
+  overflow: visible; /* For IE */
+  padding: 0 15px;
+  user-select: none;
+  width: max-content !important;
+  height: 14px;
+  width: 83px;
+  color: #949CA2;
+  font-family: inherit;
+  font-size: 13px;
+  font-weight: bold;
+  letter-spacing: 0.3px;
+  line-height: 14px;
+`;
+
+const HorizontalRule = (props) => (
+  <div contentEditable={false} {...props.attributes}>
```

Comment:
In regards to the error @Michael-Grover saw, maybe adding `user-select: none` in the style of this div would prevent it? according to this issue: https://github.com/ianstormtaylor/slate/issues/3421